modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/BoundObjectDefinitionsExportImportTest.java

```diff
@@ -0,0 +1,280 @@
+@FeatureFlags("LPS-187142")
+@RunWith(Arquillian.class)
+public class BoundObjectDefinitionsExportImportTest
+	extends BaseExportImportTestCase {
+
+	@ClassRule
+	@Rule
+	public static final LiferayIntegrationTestRule liferayIntegrationTestRule =
+		new LiferayIntegrationTestRule();
+
+	@Before
+	public void setUp() throws Exception {
+		user = TestPropsValues.getUser();
+
+		ObjectDefinitionResource.Builder builder =
+			_objectDefinitionResourceFactory.create();
+
+		_objectDefinitionResource = builder.user(
+			user
+		).build();
+
+		Class<?> clazz = getClazz();
+
+		_defaultDraftObjectDefinitionJSONString = StringUtil.read(
+			clazz.getResourceAsStream(
+				"dependencies/test-default-draft-object-definition.json"));
+		_defaultObjectRelationshipJSONString = StringUtil.read(
+			clazz.getResourceAsStream(
+				"dependencies/test-default-object-relationship.json"));
+	}
+
+	@Test
+	public void testExportImportBoundObjectDefinitions() throws Exception {
+		String draftBoundObjectDefinitionsJSONString = String.valueOf(
+			JSONUtil.putAll(
+				_jsonFactory.createJSONObject(
+					_defaultDraftObjectDefinitionJSONString
+				).put(
+					"externalReferenceCode", "TESTOBJECTDEFINITION1"
+				).put(
+					"name", "TestObjectDefinition1"
+				).put(
+					"objectRelationships",
+					JSONUtil.put(
+						_createOneToManyObjectRelationship(
+							"TESTOBJECTDEFINITION1", "TESTOBJECTDEFINITION2",
+							"TestObjectDefinition2", "objectRelationship1"))
+				).put(
+					"rootObjectDefinitionExternalReferenceCode",
+					"TESTOBJECTDEFINITION1"
+				),
+				_jsonFactory.createJSONObject(
+					_defaultDraftObjectDefinitionJSONString
+				).put(
+					"externalReferenceCode", "TESTOBJECTDEFINITION2"
+				).put(
+					"name", "TestObjectDefinition2"
+				).put(
+					"objectRelationships",
+					JSONUtil.put(
+						_createOneToManyObjectRelationship(
+							"TESTOBJECTDEFINITION2", "TESTOBJECTDEFINITION3",
+							"TestObjectDefinition3", "objectRelationship2"))
+				).put(
+					"rootObjectDefinitionExternalReferenceCode",
+					"TESTOBJECTDEFINITION1"
+				),
+				_jsonFactory.createJSONObject(
+					_defaultDraftObjectDefinitionJSONString
+				).put(
+					"externalReferenceCode", "TESTOBJECTDEFINITION3"
+				).put(
+					"name", "TestObjectDefinition3"
+				).put(
+					"rootObjectDefinitionExternalReferenceCode",
+					"TESTOBJECTDEFINITION1"
+				)));
+
+		JSONArray expectedDraftBoundObjectDefinitionsJSONArray =
+			_jsonFactory.createJSONArray(draftBoundObjectDefinitionsJSONString);
+
+		for (int i = 1; i <= 2; i++) {
+			JSONObject objectDefinitionJSONObject =
+				expectedDraftBoundObjectDefinitionsJSONArray.getJSONObject(i);
+
+			JSONArray objectFieldsJSONArray =
+				objectDefinitionJSONObject.getJSONArray("objectFields");
+
+			objectFieldsJSONArray.put(
+				_jsonFactory.createJSONObject(
+				).put(
+					"name",
+					StringBundler.concat(
+						"r_objectRelationship", i, "_c_testObjectDefinition", i,
+						"Id")
+				));
+		}
```

Comment:
I prefer that you add one by one instead of creating a loop of 2 iterations that start from the index 1. 
It is easier to add one by one than to understand the loop.
Can you refactor it, please?

You could create a method in the base so we can reuse the method here and in the other class.

Reply:
Okay!